content/kubermatic/master/tutorials-howtos/external-clusters/_index.en.md

```diff
@@ -59,16 +67,34 @@ Provisioning state depicts that the cluster is getting created:
 ![External Cluster Provisioning State](/img/kubermatic/master/tutorials/external_clusters/provisioning_status.png "External Cluster Provisioning State")
 
 Reconciling state depicts that the cluster is getting upgraded:
-![External Cluster Reconciling State](/img/kubermatic/master/tutorials/external_clusters/aks_reconcile.png "External Cluster Reconciling State")
+![External Cluster Reconciling State](/img/kubermatic/master/tutorials/external_clusters/reconciling_status.png "External Cluster Reconciling State")
+
+Deleting state depicts that the cluster is getting deleted:
+
+![External Cluster Delete State](/img/kubermatic/master/tutorials/external_clusters/aks_deleting.png "External Cluster Delete State")
+
+## Delete Cluster:
+
+{{% notice info %}}
+Delete operation is not allowed for imported clusters.
+{{% /notice %}}
+
+Cluster can be  Deleted by clicking on the delete icon next to the cluster you want to delete or from the cluster details page, which will delete and disconnect the cluster from the provider.
 
-### Deleted Cluster
+![Delete External Cluster](/img/kubermatic/master/tutorials/external_clusters/delete_externalcluster.png "Delete External Cluster")
 
-If you delete the cluster from the provider, the state in KKP will be shown as `Deleting`.
-![External Cluster Delete State](/img/kubermatic/master/tutorials/external_clusters/delete_status.png "External Cluster Delete State")
+![Delete External Cluster on Details Page](/img/kubermatic/master/tutorials/external_clusters/delete_disconnect_page.png "Delete External Cluster on Details Page")
 
-You can `Disconnect` the deleted cluster by clicking on the disconnect icon next to the cluster you want to disconnect, which will delete KKP cluster object for this cluster.
+## Disconnect Cluster
+
+{{% notice info %}}
+Disconnect operation does not delete the cluster from the cloud provider.
+{{% /notice %}}
 
-![Disconnect Deleted External Cluster](/img/kubermatic/master/tutorials/external_clusters/disconnect_deleted_cluster.png "Disconnect Deleted External Cluster")
+You can `Disconnect` an external cluster by clicking on the disconnect icon next to the cluster you want to disconnect or from the cluster details page, which will delete KKP cluster object.
```

Comment:
`which will delete KKP cluster object` - I would maybe rather say `internal cluster object in KKP`